proposals/nnnn-android-studio-sdl-plugin.md

```diff
@@ -0,0 +1,59 @@
+# Android Studio IDE SDL Plugins
+
+* Proposal : SDL -nnnn 
```

Comment:
Please use the exact template header here:
```
* Proposal: [SDL-NNNN](NNNN-filename.md)
* Author: [SDL Developer](https://github.com/smartdevicelink)
* Status: **Awaiting review**
* Impacted Platforms: [Core / iOS / Android / Web / RPC / Protocol]
```
just with the platforms changed to be only android.